src/diff/index.js

```diff
@@ -165,9 +165,7 @@ export function diff(
 			tmp = c.render(c.props, c.state, c.context);
 			let isTopLevelFragment =
 				tmp != null && tmp.type == Fragment && tmp.key == null;
-			newVNode._children = toChildArray(
-				isTopLevelFragment ? tmp.props.children : tmp
-			);
+			newVNode._children = isTopLevelFragment ? tmp.props.children : tmp;
```

Comment:
Should this property be renamed to `_renderResult` since it isn't guaranteed to be an array until after `diffChildren` is complete? I vaguely remember this confusing people before. I was hesitant to do this immediately cuz it adds yet another property to VNode and this one is particularly short lived (it's just a way to pass the result into `diffChildren`)